templates/pages/links.html

```diff
@@ -8,6 +8,13 @@
 		<p>We gather useful links to tutorials, good practices, etc. </p>
 
 	</div>
-
+<div>
```

Comment:
It's usually good practice to have consistent indentation. In this line, I'd recommend to use one tab for indentation, just like for the `<div>` above. This doesn't affect functionality, but it just makes the code easier to read :)